cmd/target.go

```diff
@@ -232,14 +226,9 @@ func NewTargetCmd(targetReader TargetReader, targetWriter TargetWriter, configRe
 
 // resolveNameProject resolves name to project
 func resolveNameProject(target TargetInterface, name string) (matches []string) {
-	tmp := KUBECONFIG
-
-	client, err := target.K8SClientToKind(TargetKindGarden)
+	gardenClientset, err := target.GardenerClient()
 	checkError(err)
-	projectLabel := "garden.sapcloud.io/role=project"
-	projectList, err := client.CoreV1().Namespaces().List(metav1.ListOptions{
-		LabelSelector: projectLabel,
-	})
+	projectList, err := gardenClientset.GardenV1beta1().Projects().List(metav1.ListOptions{})
```

Comment:
I think it would be better whether we have `*` in the project - if that is the case we can list them all. But if it is just a exact name, we could directly get it by name.